demos/food-and-beverage/supplier-distributor-with-control/resources/deliveries.gql

```diff
@@ -1,5 +1,5 @@
 mutation AddDelivery {
```

Comment:
Unrelated to this file, but related to this demo and other READMEs.  There are several places where we instruct a user to attempt to update something to which they do not have permission.  I believe nothing changes to those examples (the same `401 Unauthorized` response should be returned).  I am curious though about the removal of `error` in all the examples.  Should an error occur (say I attempt an update with an `id` value that's invalid), how will the user of these demos/examples know?

Reply:
`error` will no longer be available with the new sync mutations. In case of a GraphQL error, the standard error response will be returned:
Response code: 200
```
{
  "data" : {},
  "errors" : [//one or more errors here]
  }
  ```
  Though, I'm not 100% sure what is returned for a 401?